pkg/doc/verifiable/presentation_jwt.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package verifiable
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/square/go-jose/v3/jwt"
+)
+
+// JWTPresClaims is JWT Claims extension by Verifiable Presentation (with custom "vp" claim).
+type JWTPresClaims struct {
+	*jwt.Claims
+
+	Presentation *rawPresentation `json:"vp,omitempty"`
+}
+
+func (jpc *JWTPresClaims) refineFromJWTClaims() {
+	raw := jpc.Presentation
+
+	if jpc.Issuer != "" {
```

Comment:
Any reason to check for not empty? Assignment will still be the same since it is string.

Reply:
`raw.Holder` could have non-empty value, but we have to re-write it from JWT `iss` claim if it's defined.

Reply:
sounds good